### ISSUE TITLE
ulimits are not parsed correctly

### ISSUE THREAD

**Issue by npars** [aws/amazon-ecs-cli#75]
Specifying ulimits in a docker-compose.yml file results in a ecs-cli parse error. The same file works correctly in docker-compose. The format is the exact same as specified by the [docker docs](https://docs.docker.com/compose/compose-file/#ulimits).

### docker-compose.yml
```
helloworld:
    image: hello-world
    ulimits:
        nproc: 65535
        nofile:
            soft: 20000
            hard: 40000
```

### Output
```
$ ecs-cli compose -f docker-compose.yml --debug up
DEBU[0000] using config file: ~/.ecs/config
DEBU[0000] Opening compose file: docker-compose.yml
DEBU[0000] Parsing the compose yaml...
ERRO[0000] Unable to convert yaml to service configs     error=Error parsing: node is not of type list map[nproc:65535 nofile:map[soft:20000 hard:40000]]
ERRO[0000] Unable to open ECS Compose Project            error=Error parsing: node is not of type list map[nproc:65535 nofile:map[soft:20000 hard:40000]]
FATA[0000] Unable to create and read ECS Compose Project  error=Error parsing: node is not of type list map[nproc:65535 nofile:map[soft:20000 hard:40000]]
```

**Comment by schickling** [comment]
Any update on this?

**Comment by schickling** [comment]
Update:

@npars seems like this package parses the `ulimits` property differently. See here: https://github.com/aws/amazon-ecs-cli/blob/a3e3eeec32c4ee7667855d16094bfaab93b862a9/ecs-cli/modules/compose/ecs/utils/convert_task_definition.go#L356

This is how you should configure it:

```yaml
  ulimits:
    - "nofile=10000:10000"
```

@uttarasridhar do you have any plans to make this more compliant with the official docker-compose syntax?

**Comment by uttarasridhar** [comment]
Apologies for the delay.
But this is addressed in #121 and released in [v0.4.0](https://github.com/aws/amazon-ecs-cli/releases/tag/v0.4.0).
